pkg/controller/operators/olm/operator.go

```diff
@@ -349,6 +398,22 @@ func (a *Operator) namespaceAddedOrRemoved(obj interface{}) {
 			}
 		}
 	}
+	// find any API services owned by this namespace and requeue them
+	stringSelector := fmt.Sprintf("%v==%v", ownerutil.OwnerNamespaceKey, namespace.GetName())
+	labelSelector, err := labels.Parse(stringSelector)
+	if err != nil {
+		logger.WithError(err).Warn("parser failed")
+		return
+	}
+	apiServices, err := a.lister.APIRegistrationV1().APIServiceLister().List(labelSelector)
+	if err != nil {
+		logger.WithError(err).Warn("APIService lister failed")
+		return
+	}
+	for _, svc := range apiServices {
+		logger.Infof("gcing api service %v", svc.GetName())
```

Comment:
nit: trying to migrate towards `logger.WithField("apiservice", svc.GetName()).Info` formatting for logs